### ISSUE TITLE
nemo-terminal does not abide by chosen user shell, root shell when not bash

### ISSUE THREAD

**Issue by modoj** [linuxmint/nemo-extensions#141]
All other terminals I open abide by my chosen shell, zsh when opened, as is dictated by /etc/passwd. Nemo-terminal is the only exception to this, as it defaults to bash no matter what. I looked through previous issues and did not see this noted, so I'm reporting it, as a terminal shell should use a chosen shell. If I'm missing some kind of setting somewhere to change nemo-terminal's preference, I apologize.

This is nothing more than a nuisance, really, as it's only when using nemo I choose to use this terminal, but it is an issue that should be corrected, in my opinion (as it is standard practice for terminal shells to abide by user-shell), although I would say a very low priority one. 

In case I'm a niche case, and it does not perform this way for others, I'm running nemo on both archlinux with the current gnome, and ubuntu 15.04 with unity, I get the same result on both.

**Comment by modoj** [comment]
I noticed this in the extension's xml, is there a reason to including bash as default, rather than leaving blank, like the description notes, so that it defaults to user shell? If nothing else depends on it being bash (nothing that I've noticed, as I run zsh every time I open nemo), then I feel it should default to user.
'''
<key name="terminal-shell" type="s">
	       <default>"/bin/bash"</default>
	        <summary>Shell executable</summary>
	        <description>Path to the shell executable nemo-terminal should use. Blank indicates shell will be user's default. Note that all other default parameters assume a bash-like shell.</description>
	    </key>
'''

**Comment by EduardDopler** [comment]
Could you try changing it on your machine? Note that after the change you should trigger recompiling the XML file via `glib-compile-schemas` and restart nemo.

**Comment by modoj** [comment]
Works as expected, that is, wonderfully! Finally it will work as a terminal should, by defaulting to user shell. I tested under Ubuntu 15.04 Unity, Arch Linux with Gnome, and I created a VM to test on Mint, all work flawlessly, regardless of user-shell chosen, bash, zsh, dash, sh, etc. If user-shell remains bash, the extension functions as it does currently. 

Also of note, all functions of the extension, such as following directory change, continue to function just as well.

 I took the liberty of creating a commit with these changes.
